README.md

```diff
@@ -24,7 +24,8 @@ const combinedComposables = composable0.compose(composable1, composable2, compos
 
 ### Descriptor
 
-**Composable descriptor** (or just **Descriptor**) is a meta data object. It's a collection of properties that contain the information necessary to create an object instance by a *composable*.
+**Composable descriptor** (or just **Descriptor**) is a meta data object. It's a collection of properties that contain
+the information necessary to create an object instance by a *composable*.
```

Comment:
If a line is 300 characters long:
* The people who have soft wraps off will **suffer**.
* The people who have soft wraps on will be *ok*.

If there are 3 lines 100 characters each:
* The people who have soft wraps off will *ok*.
* The people who have soft wraps on will be *ok*.

Basically, everybody wins if we use hard wraps in our code. In fact, I have never seen an open source project which does not limit line length. Probably 99% of projects are like that.

This bloody discussion is so annoying...